__tests__/data/area_offices_data_test.js

```diff
@@ -0,0 +1,31 @@
+import fs from "fs";
+
+describe("Area offices data", () => {
+  let data;
+  beforeEach(() => {
+    data = JSON.parse(fs.readFileSync("data/data.json")).areaOffices;
+  });
+
+  it("is not empty", () => {
+    expect(data).not.toHaveLength(0);
+  });
+
+  it("has required fields in all rows", () => {
+    data.forEach(a => {
+      expect(
+        !!a.address_en &&
```

Comment:
This `!!` turns the string into a boolean. Without it when you && everything together you end up with a string rather than a boolean.